pkg/cli/klothomain.go

```diff
@@ -410,31 +408,30 @@
 		Configuration:    &appCfg,
 		OutputOptions:    options.Output,
 	}
-
+	provider, err := providers.GetProvider(&appCfg)
+	if err != nil {
+		return err
+	}
+	kb, err := providers.GetKnowledgeBase(&appCfg)
+	if err != nil {
+		return err
+	}
+	engine := engine.NewEngine(provider, kb)
```

Comment:
nit: this var is shadowing the `engine` import.

Reply:
updating in my new pr